doc/nrf/releases/release-notes-1.8.0.rst

```diff
@@ -0,0 +1,823 @@
+.. _ncs_release_notes_180:
+
+|NCS| v1.8.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+Wireless protocol stacks and libraries may be experimental if they are under development or based on verification and certification status.
+Labels in Kconfig indicate experimental configurations and can generate warnings in build output if they are used in a project.
+These configurations should not be used in production.
+See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
+* Remote procedure call library (:ref:`ble_rpc`) has full API support to interface the Bluetooth® LE Host on another CPU core.
+* The :ref:`central_and_peripheral_hrs` sample has been added to demonstrate how to use Bluetooth® with Central and Peripheral roles concurrently.
+* The :ref:`nrf_modem` for nRF9160 has been updated to version 1.4.1 with improvements and support for Modem version 1.3.1.
+* Wi-Fi coexistance with Bluetooth LE and LTE coexistance with Bluetooth LE have been added for the nRF52 family.
```

Comment:
coexistance => coexistence

```suggestion
* Wi-Fi coexistence with Bluetooth LE and LTE coexistence with Bluetooth LE have been added for the nRF52 family.
```

Reply:
Note: Here we state "Wi-Fi" and "LTE" coexistence, but further down we're more generic and use 3-wire and 1-wire, respectively. The phrasing should probably be aligned (@emidttun).

Reply:
Discussed and aligned through Teams.